parcel_test.go

```diff
@@ -31,57 +31,115 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	require.NoError(t, err)
+
+	defer db.Close()
+
 	store := NewParcelStore(db)
-	parcel := getTestParcel()
+	service := NewParcelService(store)
 
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	testParcel := getTestParcel()
+	id, err := service.store.Add(testParcel)
```

Comment:
Это нужно везде поправить
```suggestion
	id, err := store.Add(testParcel)
```